.github/workflows/publish-spm.yaml

```diff
@@ -60,7 +60,7 @@ jobs:
       - name: Build bdk-ffi for aarch64-apple-ios-sim
         working-directory: build
         run: |
-          cargo +nightly build --release -Z build-std --target aarch64-apple-ios-sim
+          cargo +nightly-2023-04-10 build --release --target aarch64-apple-ios-sim
```

Comment:
I don't want to hold up the release but can I ask why we're not using the `release-smaller` profile here?

Reply:
Good suggestion, originally the release-smaller didn't work with nightly, but I just double check and it now seems to work. I'll update the workflow and retest it. If it works now will update this PR before merging it.

Reply:
btw, enabling the `release-smaller` profile for `aarch64-apple-ios-sim` makes the final xcframework zip file about 20MB smaller, and every little bit helps!